.buildkite/pipeline.yml

```diff
@@ -53,6 +53,8 @@ steps:
   - label: E2E Tests on {{matrix}}
     key: e2e_tests
     command: |
+      # Ensure DEV_BUILD is available to npm commands
+      export DEV_BUILD=true
```

Comment:
We disable Sentry.init for E2E tests in the NodeJS initialization, But I see that DEV_BUILD removes the Sentry plugin from webpack. 

https://github.com/Automattic/studio/blob/7274053761551b6f3d2b2c91b48398b1ac29d52a/src/index.ts#L47-L48

Reply:
Unfortunately, the E2E flag only prevents Sentry from being initialized, the source maps are still uploaded, and a new hash-like release is created.
Any action on Sentry that isn't related to a known release will create a new one.